### PR TITLE
GoogleCalendarPublishAgent] Update google-api-client dependency to fi…

### PR DESCRIPTION
**Pull request by Xaelias**
…x broken API call (also had to update google-cloud-translate, faraday and oauth2)

Fixes https://github.com/huginn/huginn/issues/2633

### CODE REVIEW

**Comment by Xaelias** [comment]
I have no idea what Travis here is trying to tell me. Halp plz??? :-D

**Comment by dsander** [comment]
It looks like the API for the translation gem changed: https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-translate

At the moment my guess is that we need to adjust the [`GoogleTranslateAgent`](https://github.com/huginn/huginn/blob/master/app/models/agents/google_translation_agent.rb) to use a similar authentication method as the [`GoogleCalendar`](https://github.com/huginn/huginn/blob/master/lib/google_calendar.rb) helper class.

File: Gemfile [hunk 1]
```diff
@@ -47,8 +47,8 @@ gem 'hypdf', '~> 1.0.10'          # PDFInfoAgent
 gem 'weibo_2', github: 'albertsun/weibo_2', branch: 'master'
 
 # GoogleCalendarPublishAgent and GoogleTranslateAgent
-gem 'google-api-client', '~> 0.13'
-gem 'google-cloud-translate', '~> 1.0.0', require: 'google/cloud/translate'
+gem 'google-api-client', '~> 0.34'
```
Comment (knu) [inline Gemfile]: Wow, we are so overdue... 🙇

Comment (knu) [reply Gemfile]: It doesn't look like Gemfile.lock is updated in sync.

Comment (Xaelias) [reply Gemfile]: Yeah I'm really not a fan of how Ruby handles dependencies if I'm completely honest...
I should some free time on Wednesday to look into the test issue and this.

**Pull request closed without merging**